library/build.gradle

```diff
@@ -28,17 +28,15 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:28.0.0'
     implementation 'com.android.support:recyclerview-v7:28.0.0'
     implementation 'com.android.support:design:28.0.0'
-    // RichUtilsKt
-    implementation 'com.github.WindSekirun:RichUtilsKt:2.4.0'
+
     // Glide
-    implementation "com.zlc.glide:webpdecoder:1.3.4.8.0"
+//    implementation "com.zlc.glide:webpdecoder:1.3.4.8.0"
```

Comment:
Why `webpdecoder` was removed?

Reply:
In the whole project as we are not using the webpdecoder. 
I have seen that you mentioned about it in your comments [GlideEx.kt](https://github.com/WindSekirun/NaraeImagePicker/blob/master/library/src/main/java/com/github/windsekirun/naraeimagepicker/utils/GlideEx.kt#L15) but didn't see that you are using it.

Reply:
As i know, No. webpdecoder will inject their functions automatically **without any code in client.** So we need test if webp / animated webp can be loading without webpdecoder.